elements/pfe-sass/variables/_3-colors.scss

```diff
@@ -73,7 +73,7 @@ $pfe-color--ui-disabled:                        $pfelements--gray-light !default
 $pfe-color--ui-disabled--hover:                 $pfelements--gray-light !default;
 $pfe-color--ui-disabled--focus:                 $pfelements--gray-light !default;
 $pfe-color--ui-disabled--visited:               $pfelements--gray-light !default;
-$pfe-color--ui-disabled--text:                  $pfelements--gray !default;
+$pfe-color--ui-disabled--text:                  $pfe-color--gray-600;
```

Comment:
This still needs the `!default` value though

Reply:
@castastrophe - I just pushed the change that adds it back.